Documentation/metrics.md

```diff
@@ -2,15 +2,15 @@
 
 **NOTE: The metrics feature is considered experimental. We may add/change/remove metrics without warning in future releases.**
 
-etcd uses [Prometheus](http://prometheus.io/) for metrics reporting in the server. The metrics can be used for real-time monitoring and debugging.
+etcd uses [Prometheus] for metrics reporting in the server. The metrics can be used for real-time monitoring and debugging.
 etcd only stores these data in memory. If a member restarts, metrics will reset.
 
 The simplest way to see the available metrics is to cURL the metrics endpoint `/metrics` of etcd. The format is described [here](http://prometheus.io/docs/instrumenting/exposition_formats/).
 
 
-You can also follow the doc [here](http://prometheus.io/docs/introduction/getting_started/) to start a Prometheus server and monitor etcd metrics.
+You can also follow the doc [here][prometeus-getting-started] to start a Prometheus server and monitor etcd metrics.
```

Comment:
Follow the [Prometheus getting started doc][prometheus-getting-started] to spin up a Prometheus server to collect etcd metrics.